src/components/CrosshairToggle.jsx

```diff
@@ -1,29 +1,36 @@
 import React, { Component } from 'react';
 import {connect} from '../store/Connect';
-import {CrosshairIcon} from './Icons.jsx';
+import {CrosshairIcon, CrosshairOffIcon, CrosshairOnIcon, CrosshairTooltipIcon} from './Icons.jsx';
```

Comment:
Is `CrosshairIcon` used anymore? If not remove it from here and also remove it from `Icons.jsx`